### ISSUE TITLE
ACCESSIBILITY: Final Wave and LightHouse audit. (currently: 96%)

### ISSUE THREAD

**Issue by Eleanorgruth** [Eleanorgruth/whats-cookin#99]
- **You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.**

- [ ] Add form label to the search 
- [ ] Missing fieldset for the radio buttons
- [ ] Any other changes needed

Score 3: All accessibility areas have been considered and tested. An accessibility audit with a score of at least 95%.
